frameworks/Go/fiber/src/server.go

```diff
@@ -1,19 +1,116 @@
 package main
 
-import "github.com/fenny/fiber"
+import (
+	"database/sql"
+	"math/rand"
+	"runtime"
+	"sort"
+	"strconv"
 
+	"github.com/fenny/fiber"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var (
+	db                *sql.DB
+	stmtWorldSelect   *sql.Stmt
+	stmtWorldUpdate   *sql.Stmt
+	stmtFortuneSelect *sql.Stmt
+)
+
+// JSON struct for json serialization
 type JSON struct {
 	Message string `json:"message"`
 }
 
+var hwJSON = JSON{"Hello, World!"}
```

Comment:
This can't be declared outside of the `jsonSerialization` function. It must happen on every request.

Reply:
The serialization still happens in the request, but I changed it it the new pull to be sure.